.dockerignore

```diff
@@ -1,5 +1,9 @@
-client/node_modules
-client/build
+# Used to trim context during during docker builds.
```

Comment:
```suggestion
# Used to trim context during docker builds.
```

Reply:
So, when you do `docker build` the only positional argument it takes is the context. The context is the working directory on the host machine. `.dockerignore` is only used during this `docker build` process.

Docker copies all the files in the context into a temporary file location and then starts executing the Dockerfile. This is useful in cases when you want to copy files from the host machine into the image during build. 

We have no reason to copy anything other than what's in server/ since all we use containers for right now is to run flask.